### DISCUSSION TITLE
Changing checkpoint version

### DISCUSSION THREAD

**Discussion by IoSonoPiero** [vladmandic/sdnext#401] [Q&A] [answered]
Hello, sorry again for the noob question.
Currently, the Stable Diffusion checkpoint is 1.5.

If I'd like to change to, i.e., Stable Diffusion 2.1, do I need to download it and put it into the "models\Stable-diffusion" folder?
On Youtube, I've seen people downloading a yaml file, too.
Do I have to do the same?

Thanks in advance for any help.

**Comment by vladmandic** [comment, marked as answer]
yes, you place downloaded model in that folder. if model comes with specific yaml config, download that and place it next to model as well.

**Comment by IoSonoPiero** [comment]
Thank you very much!
